src/js/models/v2/awardsV2/BaseAwardRecipient.js

```diff
@@ -3,16 +3,16 @@
  * Created by David Trinh 10/9/18
  */
 
-
 import CoreLocation from 'models/v2/CoreLocation';
 
 const BaseAwardRecipient = {
     populate(data) {
         this.internalId = data.recipient_hash || '';
         this.name = data.recipient_name || 'Unknown';
         this.duns = data.recipient_unique_id || '--';
-        this.parentName = data.recipient_parent_name || '--';
+        this.parentName = data.parent_recipient_name || '';
```

Comment:
Did we add this field to the mock API?

Reply:
Added to the mock API response we use in unit tests. It's also been added to the [API Contract](https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/api_contracts/contracts/awards/AwardProfile.md#recipient-object).